controller/src/beerocks/master/db/db.h

```diff
@@ -226,6 +226,8 @@ class db {
     std::set<std::string> get_node_siblings(std::string mac, int type = beerocks::TYPE_ANY);
     std::set<std::string> get_node_children(std::string mac, int type = beerocks::TYPE_ANY,
                                             int state = beerocks::STATE_ANY);
+    std::string get_ruid_key_from_parent_mac_and_ruid(std::string parent_mac,
```

Comment:
this commit is redundant as is this function - just do `parent_mac + "_" + ruid` when you need the key. It is much more clear this way that this is the key. Then add a separate commit which deletes this function completely.

Reply:
I think it should be in function. It doesn't seems right to me to concatenate 2 strings in all the places which you have to use key to access a node.
Besides, If one day we want to change the key, we will have to do it in a lot of files.
I think that it is good idea maybe to change the function name to:
 std::string get_node_key(const std::string &al_mac, const std::string &ruid)

Reply:
sounds good - do that then.